airbyte-integrations/connectors/source-google-ads/source_google_ads/google_ads.py

```diff
@@ -80,7 +82,10 @@ def convert_schema_into_query(
 
         if cursor_field:
             # Fix issue 5411: Make date_start and date_end inclusive.
```

Comment:
is this comment still relevant?

Reply:
#5411 was implemented, merged and closed.